app/assets/javascripts/darkswarm/controllers/pageset_ctrl.js.coffee

```diff
@@ -0,0 +1,6 @@
+Darkswarm.controller "PagesetCtrl", ($scope, $location) ->
+  $scope.selected = ->
+    $location.path()[1..] || $scope.defaultPage
```

Comment:
some magic going on here: $location.path()[1..]
can you extract to method and give it a good name?

Offering some alternatives to the names if you like these better:
- selectedPage instead of selected
- selectDefaultPage instead of selectDefault
- PageSelectionCtrl instead of PagesetCtrl